cont.c

```diff
@@ -1547,55 +1610,15 @@ cont_restore_thread(rb_context_t *cont)
     }
 }
 
-NOINLINE(static void fiber_setcontext(rb_fiber_t *new_fiber, rb_fiber_t *old_fiber));
 
-static void
-fiber_setcontext(rb_fiber_t *new_fiber, rb_fiber_t *old_fiber)
+static rb_context_t *
+cont_ptr(VALUE obj)
```

Comment:
Could you place before `cont_save_machine_stack` in the same order?
Shuffling functions makes this harder to read.

Reply:
Apologies, I didn't mean to re-order these - just got a bit mixed up.